src/update-manager.js

```diff
@@ -138,7 +138,7 @@ export default class UpdateManager {
           var status = kb.each(response, this.ns.http('status'))
           if (status.length) {
             for (let i = 0; i < status.length; i++) {
```

Comment:
Values of rdf nodes are strings not integers

Reply:
Ah of course - good spot. Very happy with #355 ;)